blog/2023-06-08-Advisory-committee-member

```diff
@@ -0,0 +1,37 @@
+
+Today OpenRefine is looking for (a) new Advisory Committee member(s) to strengthen our governance and continue to engage at an organizational level with our partners and community.
+
+With the support of the Project Manager, OpenRefine's Advisory Committee oversees the project and coordinates with Code for Science & Society for its operations. 
+Represent OpenRefine with Code for Science and Society
+* Prepare and approve budgets
+* Approve grant application 
+* Oversee hiring and management of contractors
+* Vote to terminate contracts when necessary.
+
+In 2023, the Advisory Committee planned the creation of the [Ambassador Council](https://forum.openrefine.org/t/proposition-to-create-openrefine-ambassador-council/462) to improve community involvement in decision-making and ensure the project’s long-term sustainability.
+
+## About the position​
+
+We are seeking at least one new Advisory Committee member to help us pursue the goals described above, bringing their perspective on how OpenRefine should evolve.
+This position is for you if you:
+* Care deeply about the future of OpenRefine;
+* Have good communication skills in English;
+* Are able to join a monthly call with Code for Science and Society where we coordinate the project's operations.
+* Are ready to invest at least 3 hours per month (between calls and document review).
+* This is a volunteer (non paid) position.
+
+## How to respond​
+
+Please nominate yourself or other members of the community by contacting advisory.committee@openrefine.org. The current advisory committee will engage a conversation with the nominees and select one or more new members. New members will be announced on our [blog](https://openrefine.org/blog) and [forum](https://forum.openrefine.org/).
+
+*OpenRefine is fiscally sponsored by Code for Science and Society (CS&S). CS&S is an equal opportunity employer committed to hiring a diverse workforce at all levels of the organization thereby creating a culture that allows us to better serve our clientele, our employees and our communities. We value and encourage the contributions of our colleagues and strive to create an environment where everyone can reach their full potential and drive outstanding results. All qualified applicants will receive consideration for employment without regard to race, national origin, age, sex, religion, disability, sexual orientation, marital status, veteran status, gender identity or expression, or any other basis protected by local, state, or federal law. This policy applies with regard to all aspects of one's employment, including hiring, transfer, promotion, compensation, eligibility for benefits, and termination.*
```

Comment:
Not sure this is worth including, since this is about CS&S as an *employer*, and this is not a job posting.

Reply:
That is a good point.